libraries/Microsoft.Bot.Connector/Authentication/TimeSpanExtensions.cs

```diff
@@ -17,11 +17,8 @@ public static class TimeSpanExtensions
         /// to the delay as random noise to distribute requests across time.
         /// </summary>
         /// <param name="delay">The requested delay.</param>
-        /// <param name="multiplier">A multiplier.</param>
         /// <returns>A uniformly distributed time span.</returns>
-#pragma warning disable CA1801 // Review unused parameters (we can't change this without breaking binary compat)
-        public static TimeSpan WithJitter(this TimeSpan delay, double multiplier = 0.1)
```

Comment:
Wouldn't it be better to use multiplier in the code instead of removing it?

Reply:
We could use it instead of the hardcoded `0.1` that multiplies the delay, but we should update the description of the method because it wouldn't be generating a distribution between 0 and 10% of the delay anymore. 
Do you see another way of using the parameter?
![image](https://user-images.githubusercontent.com/44245136/152600485-b4f096a0-4c99-4a8a-a4c0-c8692e9b4f04.png)

Reply:
Either way would be ok. Simply removing it will work.